components/Home/HomeDownloadButton.tsx

```diff
@@ -1,41 +1,44 @@
 import LocalizedLink from '../LocalizedLink';
 import { useNextraContext } from '../../hooks/useNextraContext';
 import { getNodejsChangelog } from '../../util/getNodeJsChangelog';
-import type { NodeVersionData } from '../../types';
+import type { NodeReleaseData } from '../../types';
 import type { FC } from 'react';
 
-type HomeDownloadButtonProps = Pick<
-  NodeVersionData,
-  'isLts' | 'node' | 'nodeMajor' | 'nodeNumeric'
->;
+type HomeDownloadButtonProps = {
+  nodeReleaseData?: NodeReleaseData;
+};
 
 const HomeDownloadButton: FC<HomeDownloadButtonProps> = ({
-  node,
-  nodeMajor,
-  nodeNumeric,
-  isLts,
+  nodeReleaseData,
 }) => {
   const {
     frontMatter: { labels },
   } = useNextraContext();
 
-  const nodeDownloadLink = `https://nodejs.org/dist/${node}/`;
-  const nodeApiLink = `https://nodejs.org/dist/latest-${nodeMajor}/docs/api/`;
-  const nodeAllDownloadsLink = `/download${isLts ? '/' : '/current'}`;
+  if (!nodeReleaseData) return null;
```

Comment:
Why? NodeReleaseData should **always** be present. It should not be an optional type!